corehq/apps/app_manager/models.py

```diff
@@ -5579,6 +5581,14 @@ def get_latest_master_release(self):
         else:
             raise ActionNotPermitted
 
+    @classmethod
+    def wrap(cls, data):
+        self = super(LinkedApplication, cls).wrap(data)
+        if not self.pulled_from_master_version:
+            self.pulled_from_master_version = self.version
```

Comment:
i think we should add a note why this is done?
my assumption is for linked applications before this change.

Reply:
sure - ebd39150e22